### ISSUE TITLE
Clearer feedback on inventory part import failure

### ISSUE THREAD

**Issue by maxbates** [autodesk-bionano/genome-designer#937]
When imprting parts from the inventory, there is a loading state (pulsing) and error state (red and disabled). There is not feedback when a part import fails.

Mostly relevant to NCBI.

related to #856

**Comment by maxbates** [comment]
@jlachoff are you ok with this or want to make a change?

**Comment by jlachoff** [comment]
Yes, we should do something in that case. I will look at it today and suggest something.

**Comment by jlachoff** [comment]
Should turn blue and pulse on hover, and cancel gray hover state once user clicks on item.
1. When you can't fetch the data at all, just leave the item displayed as normal, but show a yellow banner with the generic message, "Unable to get the requested information from the database. Learn more…"
2. When you CAN fetch the data, but cannot parse it, that's the time to use the red text style for sure, (current behavior), but should also show a yellow banner, text, "The data is in a format we could not understand. Learn more…"
